applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -32,22 +32,26 @@ config AUDIO_FRAME_DURATION_US
 
 choice AUDIO_SAMPLE_RATE
 	prompt "Audio sample rate"
-	default AUDIO_SAMPLE_RATE_48000_HZ if BT_AUDIO_BROADCAST_RECOMMENDED || BT_AUDIO_UNICAST_RECOMMENDED
+	default AUDIO_SAMPLE_RATE_48000_HZ
 	default AUDIO_SAMPLE_RATE_16000_HZ if BT_AUDIO_BROADCAST_16_2_1 || BT_AUDIO_BROADCAST_16_2_2 || BT_AUDIO_UNICAST_16_2_1
 	default AUDIO_SAMPLE_RATE_24000_HZ if BT_AUDIO_BROADCAST_24_2_1 || BT_AUDIO_BROADCAST_24_2_2 || BT_AUDIO_UNICAST_24_2_1
 	help
 	  This will be selected based on the BAP settings.
 
 config AUDIO_SAMPLE_RATE_16000_HZ
 	bool "16 kHz"
-	depends on AUDIO_SOURCE_I2S
+	help
```

Comment:
What was the reason for removing "depends on"?
I might see it when I look at the other files

Reply:
It was to expose the #error if a user tries to compile with 16kHz and USB. If it was dependent on I2S and you try to set sample_rate to 16 or 24, it would just change to 48kHz with no error, just a warning in the compile message far up.